disperser/dataapi/docs/docs.go

```diff
@@ -353,17 +353,20 @@ const docTemplate = `{
             "type": "object",
             "properties": {
                 "commitment": {
-                    "$ref": "#/definitions/core.Commitment"
```

Comment:
Feel like these changes are not relevant to this PR? Could we get them merged separately?

Reply:
Yep I got it but it becomes confusing to review because it's hard to tell what to focus on.